### PR TITLE
Fixed bug + log levels from previous commit

### PR DESCRIPTION
**Pull request by angieenee**
Previously, this branch was failing travis at the criu check stage due to a minor bug. This commit also includes additional logging changes. 

Signed-off-by: Angie Ni <avtni@google.com>

### CODE REVIEW

**Comment by angieenee** [comment]
Question: what are best practices for merging these changes to the upstream PR? Should I squash and merge?

**Comment by kashyapp** [comment]
Could you explain what the issue was? I see some perrors changed to perr and some newlines added and removed.

**Comment by mclapinski** [comment]
@kashyapp `ret = kerndat_try_load_cache();` this line left the `ret` at a value bigger than 0. The `ret = 0;` fixes that.

**Comment by mclapinski** [comment]
@angieenee I don't know what you mean by merge here. I think you should squash and force-push.

**Review by mclapinski — CHANGES_REQUESTED** [review]
Do
```
git checkout 0016-wip
git diff criu-dev
```
There are some whitespace errors (like unexpected tabulations) I'd like you to fix.

**Comment by kashyapp** [comment]
> @kashyapp `ret = kerndat_try_load_cache();` this line left the `ret` at a value bigger than 0. The `ret = 0;` fixes that.
Ah, I missed that line.

**Comment by kashyapp** [comment]
> @kashyapp `ret = kerndat_try_load_cache();` this line left the `ret` at a value bigger than 0. The `ret = 0;` fixes that.

So that was why the original code had `ret = check_pagemap()`. Thanks. In trying to make it consistent, we broke it.

File: criu/kerndat.c [hunk 1]
```diff
@@ -1053,66 +1066,126 @@ int kerndat_init(void)
 
 	preload_socket_modules();
 	preload_netfilter_modules();
+	ret = 0;
```
Comment (kashyapp) [inline criu/kerndat.c]: move this under 
```
if (ret <= 0)
		return ret;
```

**Review by mclapinski — APPROVED** [review]
nice work on the whitespace differences

File: criu/kerndat.c [hunk 2]
```diff
@@ -653,6 +659,7 @@ static int kerndat_detect_stack_guard_gap(void)
 
 	maps = fopen("/proc/self/maps", "r");
 	if (maps == NULL) {
+		pr_perror("Could not open /proc/self/maps.\n");
```
Comment (mclapinski) [inline criu/kerndat.c]: remove newline

Comment (mclapinski) [reply criu/kerndat.c]: and the dot for all pr_perror

**Pull request merged**